controller.go

```diff
@@ -31,6 +31,7 @@ const (
 	controllerAgentName = "cronjob-controller"
 	intTrue             = 1
 	searchLabel         = "controller-uid"
+	slackAnnotationName = "slack-channel"
```

Comment:
It seems like common practice to namespace annotations to the tool that is expected to read them.
```suggestion
	slackAnnotationName = "kube-job-notifier/slack-channel"
```

Reply:
Also as existing behaviour is to have a different channel per success or failure perhaps it would be good to have a way to set different channels per job.

e.g.

`kube-job-notifier/slack-channel-success` or `kube-job-notifier/slack-channel-start` or `kube-job-notifier/slack-channel-failed`